### DISCUSSION TITLE
nova component not updating with fresh (updated) data.

### DISCUSSION THREAD

**Discussion by smcnepal** [laravel/nova-issues#4465] [Q&A]
- Laravel Version: 8.1
- Nova Version: 3.20
- PHP Version: 7.4
- Database Driver & Version:
- Operating System and Version: windows 10
- Browser type and version: chrome latest, same issue with firefox.

**Comment by mateusgalasso** [comment]
Did you cleaned the views?

**Reply by smcnepal** [reply]
yes

**Reply by smcnepal** [reply]
didn't get any changes after cleaning view,config,cache,route and composer and npm reinstall also.

**Reply by mateusgalasso** [reply]
Are using some package? I have this issues on Nova Tabs

**Reply by smcnepal** [reply]
yes...

**Reply by mateusgalasso** [reply]
You could add a issue there. 
For now I'm working around, redirecting to the same page

**Reply by baszach** [reply]
@stonkeep @smcnepal  

I have the same issue. How could this be connected to Nova Tabs? Do you have a link to an issue in the Nova Tabs repository?

How does your work-around work :) ??

**Reply by smcnepal** [reply]
@baszach please add the issue to the  laravel/nova-issue
github repo.

**Reply by baszach** [reply]
You already did and it was turned into a discussion. Furthermore I'm still investigating if it's my issue or Nova related.
